src/System.Net.HttpListener/tests/WebSocketTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Http;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    public class WebSocketTests
+    {
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        public static async Task AcceptWebSocketAsync_NullSubProtocol_Succeeds()
+        {
+            string url = UrlPrefix.CreateLocal();
```

Comment:
Yeah.. all the listeners will be using the same socket, but listen to different url prefixes.. So multiple httplisteners will not send response for the same request.